src/core/ComplexFilter/index.tsx

```diff
@@ -71,50 +75,17 @@ const ComplexFilter = <Multiple extends boolean | undefined = false>({
     }
   }, [propValue]);
 
-  // * (mlila): likely, this portion on ComplexFilter will need to be replaced with Dropdown (or a
-  // * new DropdownFilter) component. As ComplexFilter evolves, there will be more types added,
-  // * such as sliders for ranges, inline multi selects, etc.
-  return (
-    <>
-      <Wrapper {...rest}>
-        <InputDropdownComponent
-          label={label}
-          onClick={handleClick}
-          sdsStage={open ? "userInput" : "default"}
-          {...InputDropdownProps}
-        />
-
-        <StyledChipsWrapper>
-          <Chips value={value} multiple={multiple} onDelete={handleDelete} />
-        </StyledChipsWrapper>
-      </Wrapper>
+  const handleClose = () => {
```

Comment:
I see that the handler functions are changed from function declaration to anonymous function expression, and moved above the return statement

Just wanted to provide some context for the code structure there 😄 --

Traditionally people prefer to declare functions before using them, but nowadays with IDEs being able to jump to function definitions easily, I've leaned towards a newer practice of putting what the component returns above the fold (what people see in IDE without scrolling down), since that's most readers care about the most, and hide the implementation details below the fold, so readers only need to scroll to find the implementation if needed

Also function declaration helps with error stack trace, since the functions aren't anonymous!

Please let me know what you think of this rationale 🙏 Thank you!

Reply:
Oh, that sounds just right. I totally agree with that. I'll revert the unnecessary changes. Thank you 🫠

Reply:
Awesome! So glad you like the rationale too 🤩 👏 I still appreciate the effort you took to refactor the code even though we're not using it this time 🙏